MozziGuts.cpp

```diff
@@ -244,6 +245,76 @@ ISR(ADC_vect, ISR_BLOCK)
 	adc_count++;
 }
 #endif // end main audio input section
+#endif
+
+
+#if IS_SAMD21()
+#define AUDIO_BIAS ((uint16_t) 2048)
+
+static bool tcIsSyncing()
```

Comment:
Can you explain a bit, what these tcXXX() functions are doing (add comments and / or use a more explanatory naming). I have half a mind to go try to clean up all the hardware-specific \#ifs one of those days, but for that purpose it would be really important to still be able to understand what's going on.

Reply:
They are the timer functions analogous to the Teensy ones which were built into a nice library. I haven't been able to navigate the SAMD21 universe well enough to see if there is a good library. The problem with the code I added is that I picked a specific timer which may conflict. I will ask Dave Mellis who is managing the conventional use of these across the libraries.

Reply:
Just add a comment along the lines of "these functions (tcXXX()) are used to run a timer at AUDIO_RATE, to copy samples for the output buffer to the DAC on SAMD21". (If I have understood their purpose, correctly). I.e. document the _why_ of these functions, somehow.

Reply:
OK